src/pages/privacy.md

```diff
@@ -55,4 +55,8 @@ encryption in your operating system.
 OpenRefine does not store user data in cookies, except for OAuth authentication,
 used when importing or exporting projects to a Google Drive, or for Wikibase authentication.
 
+## Update notifications on OpenRefine's start page
 
+To detect whether a new version is available, OpenRefine makes web queries to external websites when its start page is opened. This can be disabled by [setting the system property](docs/manual/running#jvm-preferences) `refine.display.new.version.notice` to `false`.
```

Comment:
```suggestion
To detect whether a new version is available, OpenRefine makes web queries to external websites when its start page is opened. This can be disabled by [setting the system property](docs/manual/running#jvm-preferences) `refine.display.new.version.notice` to `false`, which is its default value unless the user has opted-in.
```

Hopefull the above is true about the default

Reply:
No, that's the contrary.

Reply:
I forgot about the historical case. I've updated my suggested text. If that disagrees with the code, let's adjust the code so that it actually is opt-in (as it should be).

Reply:
That's still not quite right! There are two things:
* the system property, which if set to `true`, disables the entire feature (including prompting the user to opt in)
* the OpenRefine preference value, which set when the user opts in

That might sound like a complicated set up but I think that's actually fitting: in environments where the user has no control over applying updates, it makes sense for the system administrator to set `refine.display.new.version.notice` to `false` so that the user doesn't get prompted at all. Then, after prompting, it makes sense to store the user's preference in the preference store indeed, so that they are able to change this choice easily.

Reply:
I'm merging this as is so that it's in place for the release, but happy to tweak this part further of course.

Reply:
OK, so on further checking, `refine.display.new.version.notice` (which actually means "don't poll for version updates") appears to have been [added in 3.6](https://openrefine.org/whats_new#new-features-1), so I've updated my suggested text to reflect that.

I've also added a suggestion to the post-3.8 privacy notice to say that it's opt-in.

Reply:
I can't apply your suggestions after merging the PR, so I have copied them over to another commit: https://github.com/OpenRefine/openrefine.org/commit/99a4fff6b2bb53ca02c492122d46ddf70cc90a57. In such a situation it'd be more convenient if you could make the commit directly (or a PR if you prefer)